src/test/scala/mesosphere/mesos/PersistentVolumeMatcherTest.scala

```diff
@@ -3,10 +3,13 @@ package mesosphere.mesos
 import mesosphere.UnitTest
 import mesosphere.marathon._
 import mesosphere.marathon.core.instance.{Instance, LocalVolumeId, TestInstanceBuilder}
-import mesosphere.marathon.state.{AppDefinition, PathId}
+import mesosphere.marathon.core.task.Task
+import mesosphere.marathon.state.AppDefinition
 import mesosphere.marathon.stream.Implicits._
 import mesosphere.marathon.test.MarathonTestHelper
 
+import scala.collection.immutable.Seq
```

Comment:
Hm, there should be no need to this since we have a `Seq` definition in `mesosphere.marathon`.